docs/reference/release-notes.md

```diff
@@ -0,0 +1,33 @@
+# COS 2 release notes
+October 2025
+
+These release notes cover new features and changes in COS 2.
+
+COS 2 is designated as a short-term support release. This means it will continue to receive security updates and critical bug fixes for 9 months.
+
+See our [release policy](release-policy.md) and [upgrade instructions](../how-to/upgrade.md).
+
+## Requirements and compatibility
+See [system requirements](system-requirements.md).
+
+COS 2 is compatible with Juju v3.6+.
+
+## What's new in COS 2
+
+- Terraform modules for COS and COS Lite.
```

Comment:
Can you link where they get these modules? Are they templates? 

I'm not that familiar with Terraform, so if there's no where sensical to link, then leave it as is.

But also - this item feels like a much bigger change than the other items in the list. Is this one more notable? If yes, can you add more information about it?

Reply:
Done.